release/device-snmp-go.yaml

```
@@ -0,0 +1,19 @@
+---
+name: 'device-snmp-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '8cef9076ddadae0bb4a2071504c212bd6ac7a64c'
```

```suggestion
commitId: 'd6a2d4438fc175be031f77f972aa414c83442272'
```
This is the final commit for the release.